docs/api/environment-variables.md

```diff
@@ -51,6 +51,18 @@ Unsupported options are:
 --http-parser
 ```
 
+The [`nodeOptions` fuse](../tutorial/fuses.md#L27) need to be enabled in order to use this variable.
```

Comment:
That fuse is enabled by default, so this should probably be
```suggestion
If the [`nodeOptions` fuse](../tutorial/fuses.md#L27) is disabled, `NODE_OPTIONS` will be ignored.
```

Same for the others below.